backend/compact-connect/tests/smoke/license_upload_smoke_tests.py

```diff
@@ -0,0 +1,153 @@
+# ruff: noqa: T201  we use print statements for smoke testing
```

Comment:
This is great! Glad we're starting to add some integration tests and appreciate how easy to follow / well-documented this code is.